stack-cmbm/recipes/deploy-nginxapp.rb

```diff
@@ -0,0 +1,101 @@
+include_recipe 'nginx-app::server'
```

Comment:
Yes there is, it is hiding in line 60. Did you miss that or are you referring to something else?